conftest.py

```diff
@@ -12,3 +16,24 @@ def pytest_configure(config):
     # XXX(dcramer): Kombu throws a warning due to transaction.commit_manually
     # being used
     warnings.filterwarnings("error", "", Warning, r"^(?!(|kombu|raven|sentry))")
+
+    # Create an empty webpack manifest file - otherwise tests will crash if it does not exist
+    os.makedirs(dist_path, exist_ok=True)
```

Comment:
I think you could move this to acceptance's conftest, right before `Running webpack to compile frontend assets - this will take awhile`. Same thing with `pytest_unconfigure`.

Reply:
There are a handful of tests in `tests/sentry` that fail if we move it: https://github.com/getsentry/sentry/actions/runs/764940332

I'm going to revert, I think it's fine to have this at the root level conftest as it only runs once, and this will prevent some frustration if someone adds a new test that would require a manifest.